src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -3,15 +3,33 @@
 
 package com.cognite.sdk.scala.v1
 
-sealed abstract class PropertyType {
+import java.time.{LocalDate, ZonedDateTime}
+// scalastyle:off number.of.types
+
+sealed abstract class DataModelProperty[TV](val value: TV)
```

Comment:
yep, Type Value... do you think it would be better to have a single letter `V`, or a more descriptive `ValueType`? (usually I would leave just `T`, but I wanted to keep it consistent with `ArrayPropertyType` bellow, and that one has a value type and a property type)

Reply:
changed it to `V` for value type and `P` for property type